zebrad/Cargo.toml

```diff
@@ -151,6 +154,7 @@ zebra-network = { path = "../zebra-network", version = "1.0.0-beta.31" }
 zebra-node-services = { path = "../zebra-node-services", version = "1.0.0-beta.31" }
 zebra-rpc = { path = "../zebra-rpc", version = "1.0.0-beta.31" }
 zebra-state = { path = "../zebra-state", version = "1.0.0-beta.31" }
+zebra-scan = { path = "../zebra-scan", version = "0.1.0-alpha.0" }
```

Comment:
This will enable the dependency in Zebra's production builds, but we only want it in experimental builds. Here's the first part of that change, I added the second part to ticket #7892.

```suggestion

# Experimental shielded-scan feature
zebra-scan = { path = "../zebra-scan", version = "0.1.0-alpha.0", optional = true }
```

Reply:
Thanks for that one.